src/include/stir/listmode/DetectorCoordinateMapFromFile.h

```diff
@@ -52,15 +52,23 @@ class DetectorCoordinateMapFromFile
 {
 public:
 	//! Constructor calls read_detectormap_from_file( filename ).
-	DetectorCoordinateMapFromFile(const std::string& filename)
+	DetectorCoordinateMapFromFile(const std::string& filename, double sigma = 1.0) :
```

Comment:
are you sure you want to default `sigma` to 1? Other places have `0`

Reply:
@KrisThielemans, yes you are right. I changed it. Travis has to run again now.